src/runtimes/docker/utils/getContainerOsString.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Return a normalized string for the OS name specifically for Docker
+ * @param os pre-normalized OS name
+ * @returns normalized OS name
+ */
+export function getContainerOSFromPlatformOS(os?: string): ContainerOS {
+    switch (os || 'Linux') {
```

Comment:
Consider normalizing the case on the string (lower or upper case) before the switch to make this re-usable to normalize values in the client argument handler as well.

Reply:
good thinking!